### DISCUSSION TITLE
Testing my Contract generate error warning

### DISCUSSION THREAD

**Discussion by cryptopunkstar** [FuelLabs/fuel-core#1598] [General]
Hi !
Please help me :)

**I coded on my terminal :**
cargo install cargo-generate

**Result 2 errors :**

error: failed to run custom build command for `openssl-sys v0.9.98`
Caused by:
  process didn't exit successfully: `/tmp/cargo-installV3DGgV/release/build/openssl-sys-dcaeb6a4a00bbfaf/build-script-main` (exit status: 101)
  --- stdout

note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
warning: build failed, waiting for other jobs to finish...
error: failed to compile `cargo-generate v0.19.0`, intermediate artifacts can be found at `/tmp/cargo-installV3DGgV`.
To reuse those artifacts with a future compilation, set the environment variable `CARGO_TARGET_DIR` to that path.

**Thank your answer !**

**Comment by BBubbyy** [comment]
Problem are solving yet?

**Comment by xgreenx** [comment]
We don't use `cargo-generate`. It seems the discussion is not relevant to the `fuel-core` repository=)

**Comment by AntiHaxx** [comment]
did you solved your problem??
